event-statistics/pom.xml

```diff
@@ -9,7 +9,6 @@
   <properties>
     <awaitility.version>4.1.1</awaitility.version>
     <compiler-plugin.version>3.8.1</compiler-plugin.version>
-    <lombok.version>1.18.22</lombok.version>
```

Comment:
Very happy to see the removal of lombok.

Reply:
Yeah I'm not much of a fan. I had it in there just to show it could work, but since now the pojo is being generated by Avro we no longer need it.

Should we introduce Lombok somewhere else in the system @cescoffier ? Even though I don't like it, it is widely used.